deploy_hpo.sh

```diff
@@ -105,6 +107,15 @@ fi
 SERVICE_STATUS_NATIVE=$(ps -u | grep service.py | grep -v grep)
 SERVICE_STATUS_DOCKER=$(${CONTAINER_RUNTIME} ps | grep hpo_docker_container)
 
+# In case of Minikube and Openshift, check if registry credentials are set as Environment Variables
+if [ ${cluster_type} == "minikube" || ${cluster_type} == "openshift" ]; then
+    if [ -z $REG_UNAME ] || [ -z $REG_PASS ] || [ -z $REG_EMAIL ]; then
```

Comment:
Please have all variables in quotes and brackets "${VAR}"

Reply:
Done